Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -352,14 +352,14 @@ bool ShouldRebuild()
         }
 
         public async Task LoadExtensionsFromDirectory(
-            DirectoryInfo directory,
+            string directory,
```

Comment:
Why the change from `DirectoryInfo`?

Reply:
Because NewtonSoft json doesn't serialize, FileInfo, DirectoryInfo.